src/css/index.css

```diff
@@ -0,0 +1,526 @@
+    height: 100%;
+    background-color: var(--banner-bg-color);
+}
+
+.banner__content-item:nth-child(1) .banner__content-background {
+    opacity: 0.6;
+}
+
+.banner__content-item:nth-child(2) .banner__content-background {
+    opacity: 0.4;
+}
+
+.banner__content-item:nth-child(3) .banner__content-background {
+    opacity: 0.4;
+}
+
+.banner__content-item:nth-child(4) .banner__content-background {
+    opacity: 0.1;
+}
+
+.banner__content-item:nth-child(4) .banner__content-text {
+    background-color: var(--button-bg-color);
+    color: white;
+}
+
+.banner__content-text {
+    position: absolute;
+    padding: 10px;
+    background-color: var(--banner-bg-color);
+    font-size: var(--h3-font);
+}
+
+/*##title*/
+
+.title {
+    margin-top: var(--default-block-margin);
+    border-bottom: 1px solid var(--title-border-color);
+}
+
+.title__content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-bottom: var(--default-block-margin);
+}
+
+.title__content-text {
+    flex-basis: 75%;
+}
+
+.title__content-main-text {
+    font-size: var(--h1-font);
+    color: var(--main-text-color);
+}
+
+.title__content-subtext {
+    font-size: var(--h3-font);
+    color: var(--sub-text-color);
+}
+
+.title__content-img {
+    font-size: var(--h3-font);
+    color: var(--button-fg-color);
+}
+
+/*##row*/
+
+.row {
+    margin-top: var(--default-block-margin);
+}
+
+.row__content {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    width: 100%;
+}
+
+.row__content-item {
+    display: flex;
+    justify-content: left;
+    align-items: center;
+    width: 50%;
+    margin: 10px auto;
+}
+
+/*rounded images*/
+.row__content-span {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 2px solid var(--row-circle-color);
+    border-radius: 130px;
+    min-height: 80px;
+    min-width: 80px;
+    width: 8vw;
+    height: 8vw;
+    text-align: center;
+}
+
+.row__content-img {
+    display: inline;
+    font-size: var(--h1-font);
+    color: var(--content-img-color);
+}
+
+.row__content-article {
+    margin: 5%;
+}
+
+.row__content-title {
+    font-size: var(--h2-font);
+    color: var(--main-text-color);
+}
+
+.row__content-text {
+    margin-top: 10px;
+    font-family: var(--sub-font);
+    font-size: var(--p-link-font);
+    color: var(--sub-text-color);
+}
+
+/*##subscribe*/
+
+.subscribe {
+    margin-top: var(--default-block-margin);
+}
+
+.subscribe__content {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    border-radius: 5px;
+    padding: 5vh 0;
+    background: var(--subscribe-bg-color);
+}
+
+.subscribe__content-text {
+    text-align: center;
+    flex-basis: 25%;
+    font-size: var(--h1-font);
+    color: var(--subscribe-title-color);
+}
+
+.subscribe__content-form {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    flex-basis: 75%;
+    margin-top: -2vh;
```

Comment:
negative marging? i'm curious what is it for.

Reply:
See an example below. If these flex blocks are wrapped, negative margin allow to suppress margin of first row and leave it only for next one.
![image](https://user-images.githubusercontent.com/45680851/78131368-a2aac780-7423-11ea-8cd0-b169935a3227.png)